test/fixtures/05-use-file-name/before.js

```diff
@@ -1,5 +1,6 @@
 import styled from "styled-components";
 
 const Test = styled.div`color: red;`;
+const before = styled.div`color: blue;`;
```

Comment:
Hm, sorry, one more thing ^^"

Can you add an "inverted" test here? A component that doesn't have the name `before`? This way we'll have both cases covered in this file